src/nodes/AssemblyStackAssignment.js

```diff
@@ -0,0 +1,12 @@
+const {
+  doc: {
+    builders: { join }
+  }
+} = require('prettier');
+
+const AssemblyAssignment = {
+  print: ({ node, path, print }) =>
+    join(' ', [path.call(print, 'expression'), '=:', node.name])
```

Comment:
Tbh this is just a modified version of what's already being done in `AssemblyAssignment`. It would make sense to split for both, but maybe we should do that in a separate PR?

Reply:
To clarify: I totally agree with the idea, I'm just trying to do the minimal amount of things we need for v1.0.0 :sweat_smile: